Policies/github-acceptable-use-policies.md

```diff
@@ -48,15 +50,17 @@ While using the Service, under no circumstances will you:
 ### 4. Services Usage Limits
 You will not reproduce, duplicate, copy, sell, resell or exploit any portion of the Service, use of the Service, or access to the Service without our express written permission.
 
-### 5. Scraping and API Usage Restrictions
-Scraping refers to extracting data from our Service via an automated process, such as a bot or webcrawler. It does not refer to the collection of information through our API. Please see Section H of our [Terms of Service](/articles/github-terms-of-service#h-api-terms) for our API Terms. You may scrape the website for the following reasons:
+### 5. Information Usage Restrictions
+You may use information from our Service for the following reasons, regardless of whether the information was scraped, collected through our API, or obtained otherwise:
+
+-  Researchers may use public, non-personal information from the Service for research purposes, only if any publications resulting from that research are open access.
```

Comment:
```suggestion
-  Researchers may use public, non-personal information from the Service for research purposes, only if any publications resulting from that research are [open access](https://en.wikipedia.org/wiki/Open_access).
```

This feels like it should be a link?

Reply:
Is `arxiv.org` considered to be open access?

Reply:
Yes.